### PR TITLE
Release 1.0.0b6

### PR DESCRIPTION
**Pull request by skarimo**
SSIA

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
 # Changelog
 
+## 1.0.0b6 / 2021-02-16
+
+* [Added] Add profile_metrics_query properties to dashboard widget requests. See [#283](https://github.com/DataDog/datadog-api-client-python/pull/283).
+* [Added] Add profile_metrics_query properties to dashboard widget requests. See [#282](https://github.com/DataDog/datadog-api-client-python/pull/282).
```
Comment (nmuesch) [inline CHANGELOG.md:6]: Whats the difference on these two?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,36 @@
 # Changelog
 
+## 1.0.0b6 / 2021-02-16
+
+* [Added] Add profile_metrics_query properties to dashboard widget requests. See [#283](https://github.com/DataDog/datadog-api-client-python/pull/283).
```
Comment (therve) [inline CHANGELOG.md:5]: I think you can remove that one, with the one below being the real change.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,36 @@
...
+* [Added] Add profile_metrics_query properties to dashboard widget requests. See [#282](https://github.com/DataDog/datadog-api-client-python/pull/282).
+* [Added] Add geomap widget to dashboards v1. See [#278](https://github.com/DataDog/datadog-api-client-python/pull/278).
+* [Added] Add v2 API for metric tag configuration. See [#277](https://github.com/DataDog/datadog-api-client-python/pull/277).
+* [Added] Add Lambda invocations usage to response. See [#276](https://github.com/DataDog/datadog-api-client-python/pull/276).
+* [Added] Remove unstable flag for logs apis. See [#268](https://github.com/DataDog/datadog-api-client-python/pull/268).
+* [Added] [monitor] Add restricted roles to update. See [#255](https://github.com/DataDog/datadog-api-client-python/pull/255).
```
Comment (therve) [inline CHANGELOG.md:11]:
```suggestion
* [Added] Add restricted roles to monitor update. See [#255](https://github.com/DataDog/datadog-api-client-python/pull/255).
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,36 @@
...
+* [Added] Add endpoint for IoT billing usage. See [#251](https://github.com/DataDog/datadog-api-client-python/pull/251).
+* [Added] Add query parameters for SLO search endpoint. See [#249](https://github.com/DataDog/datadog-api-client-python/pull/249).
+* [Added] Add fields for formula and function query definition and widget formulas. See [#245](https://github.com/DataDog/datadog-api-client-python/pull/245).
+* [Added] Add global_time to time_window slo widget. See [#243](https://github.com/DataDog/datadog-api-client-python/pull/243).
+* [Added] [MA-1666][slo correction] Update required fields in create and update requests . See [#235](https://github.com/DataDog/datadog-api-client-python/pull/235).
```
Comment (therve) [inline CHANGELOG.md:16]:
```suggestion
* [Added] Update required fields in create and update SLO correction requests . See [#235](https://github.com/DataDog/datadog-api-client-python/pull/235).
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,36 @@
...
+* [Added] ✨[LA-579][Logs] Add docs for log index creation (POST). See [#232](https://github.com/DataDog/datadog-api-client-python/pull/232).
```
Comment (therve) [inline CHANGELOG.md:17]:
```suggestion
* [Added] Add docs for log index creation. See [#232](https://github.com/DataDog/datadog-api-client-python/pull/232).
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,36 @@
...
+* [Added] [MA-1665] Add Slo Corrections to open api spec. See [#226](https://github.com/DataDog/datadog-api-client-python/pull/226).
```
Comment (therve) [inline CHANGELOG.md:18]:
```suggestion
* [Added] Add SLO Corrections. See [#226](https://github.com/DataDog/datadog-api-client-python/pull/226).
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,36 @@
...
+* [Fixed] Add missing tlsVersion and minTlsVersion to Synthetics assertion types. See [#284](https://github.com/DataDog/datadog-api-client-python/pull/284).
+* [Fixed] Rule: all nested objects in arrays must be defined on top-level. See [#279](https://github.com/DataDog/datadog-api-client-python/pull/279).
+* [Fixed] Add support for DD_SITE in examples. See [#271](https://github.com/DataDog/datadog-api-client-python/pull/271).
+* [Fixed] [dashboards] Change analyzed_spans -> spans. See [#270](https://github.com/DataDog/datadog-api-client-python/pull/270).
```
Comment (therve) [inline CHANGELOG.md:22]:
```suggestion
* [Fixed] Change dashboards analyzed_spans to spans. See [#270](https://github.com/DataDog/datadog-api-client-python/pull/270).
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,36 @@
...
+* [Fixed] Fix AWS tag filter delete request. See [#266](https://github.com/DataDog/datadog-api-client-python/pull/266).
+* [Fixed] [dashboards] Remove an unnecessary field from TimeSeriesFormulaAndFunctionEventQuery. See [#265](https://github.com/DataDog/datadog-api-client-python/pull/265).
```
Comment (therve) [inline CHANGELOG.md:24]:
```suggestion
* [Fixed] Remove an unnecessary field from TimeSeriesFormulaAndFunctionEventQuery. See [#265](https://github.com/DataDog/datadog-api-client-python/pull/265).
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,36 @@
...
+* [Fixed] Fix unit format in SLO history response. See [#260](https://github.com/DataDog/datadog-api-client-python/pull/260).
+* [Fixed] [formulas and functions ] Change group_by from object -> list of objects. See [#259](https://github.com/DataDog/datadog-api-client-python/pull/259).
```
Comment (therve) [inline CHANGELOG.md:26]:
```suggestion
* [Fixed] Change dashboards group_by from object to list of objects. See [#259](https://github.com/DataDog/datadog-api-client-python/pull/259).
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,36 @@
...
+* [Fixed] [monitor] Fix location of restricted roles. See [#254](https://github.com/DataDog/datadog-api-client-python/pull/254).
```
Comment (therve) [inline CHANGELOG.md:27]:
```suggestion
* [Fixed] Fix monitor location of restricted roles. See [#254](https://github.com/DataDog/datadog-api-client-python/pull/254).
```

**Review by therve — CHANGES_REQUESTED** [review]
Some minor nits, thanks.

File: CHANGELOG.md (hunk 1 above)
Comment (skarimo) [reply CHANGELOG.md:6]: We can remove 283, looks like its client regeneration. Only thing updated was timestamp.

**Review by therve — APPROVED** [review]

**Pull request merged**